### ISSUE TITLE
Shadow fiends first coil doesn't apply slow

### ISSUE THREAD

**Issue by mihaitimotin** [ValveSoftware/Dota2-Gameplay#8780]
### Ability name

shadow raze

### Description

Shadow fiends first coil doesn't apply slow, ability says 15% slow on level 4 but the slow is 0%, slow apply only after second coil

### Example Match ID (and possibly Timestamp)

dasdf

**Comment by Daar543** [comment]
I think this was in the original patch notes that you need to stack them. But probably the tooltip is not clear enough.

**Comment by TheUnholyWrath** [comment]
[Debuff now also applies a stackable 30% turn speed slow and 15% movement speed slow per consecutive stack (i.e: starting with the second one)](https://www.dota2.com/patches/7.32)

Adds a stacking damage amplifier on the target that causes the enemy to take bonus Shadowraze damage per stack. Each consecutive stack also decreases movement speed.

Its stated in the patch notes and tooltip how this works. If you think the tooltip isn't clear enough, please remake this issue or create a new issue. Otherwise, you can close this issue.
